packages/capabilities/src/types.ts

```diff
@@ -85,6 +85,7 @@ export interface InvalidProvider extends Ucanto.Failure {
 export type CustomerGet = InferInvokedCapability<typeof CustomerCaps.get>
 export interface CustomerGetSuccess {
   did: AccountDID
+  subscriptions: string[]
```

Comment:
It's a good question because I do think there's a fair amount of overlap, but the way I think about it at the moment is that a "provision" is the thing we create when a user invokes [`provider/get`](https://github.com/web3-storage/specs/blob/main/w3-provider.md#providerget) that can be attached to a "consumer" with [`consumer/add`](https://github.com/web3-storage/specs/blob/main/w3-provider.md#consumeradd) to create a "subscription"

In practice we've combined these two capabilities into `provider/add` so we don't actually have provisions without corresponding subscriptions, and provisions themselves don't really show up anywhere in the code (imho a good thing because I think "provision" is an awkward noun here).

I do think it makes more sense to think of a customer's "subscriptions" than their "provisions" in general, so I think this is the right name.